src/cirrus/environment.py

```diff
@@ -24,12 +34,21 @@ def cirrus_home():
         return os.environ['CIRRUS_HOME']
     
     home = inspect.getsourcefile(cirrus)
-    # from the cirrus init py in the venv, we need to
-    # move up 5 dirs to get the install directory
-    for i in range(6): 
-        home = os.path.dirname(home)
+    if ('venv' in home) and ('site-packages' in home):
+        # we are in a pip installed virtualenv site-packages
+        # from the cirrus init py in the venv, we need to
+        # move up 5 dirs to get the install directory
```

Comment:
Not clear on this. What's an example of falling into this path?
Would the cirrus import have to be imported from a pip install to include site-packages and venv in its path?